specs/lib/triBoxOverlapSpec.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+var Cesium = require('cesium');
+var Cartesian3 = Cesium.Cartesian3;
+var triBoxOverlap = require('../../lib/triBoxOverlap');
+
+describe('triBoxOverlap', function() {
```

Comment:
Are these my original tests?  How is test coverage?  It is important that we test all the case and combinations since we are going to build on this.